controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -284,6 +319,63 @@ func (r *StorageConsumerReconciler) reconcileCephBlockPool() error {
 	return nil
 }
 
+func (r *StorageConsumerReconciler) reconcileCephFilesystemSubVolumeGroup() error {
+
+	cephFilesystemList := rookCephv1.CephFilesystemList{}
+	if err := r.Client.List(r.ctx, &cephFilesystemList, client.InNamespace(r.namespace)); err != nil {
+		return fmt.Errorf("Error fetching CephFilesystemList. %+v", err)
+	}
+
+	var cephFileSystemName string
+	availableCephFileSystems := len(cephFilesystemList.Items)
+	if availableCephFileSystems == 0 {
+		return fmt.Errorf("No CephFileSystem found in the cluster")
+	} else if availableCephFileSystems > 1 {
+		klog.Warningf("More than one CephFileSystem found in the cluster, selecting the first one")
+	}
+
+	cephFileSystemName = cephFilesystemList.Items[0].Name
+
+	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, r.cephFilesystemSubVolumeGroup, func() error {
+		if err := r.own(r.cephFilesystemSubVolumeGroup); err != nil {
+			return err
+		}
+
+		r.cephFilesystemSubVolumeGroup.Spec = rookCephv1.CephFilesystemSubVolumeGroupSpec{
+			FilesystemName: cephFileSystemName,
+		}
+		return nil
+	})
+
+	cephFilesystemSubVolumeGroupDetails := &ocsv1alpha1.CephResourcesSpec{
+		Kind: r.cephFilesystemSubVolumeGroup.Kind,
+		Name: r.cephFilesystemSubVolumeGroup.Name,
+		CephClients: map[string]string{
+			"provisioner": r.cephClientCephFSProvisioner.Name,
+			"node":        r.cephClientCephFSNode.Name,
+		},
+	}
+	if r.cephFilesystemSubVolumeGroup.Status != nil {
+		cephFilesystemSubVolumeGroupDetails.Phase = string(r.cephFilesystemSubVolumeGroup.Status.Phase)
+	}
+
+	r.updateCephResourceStatus(cephFilesystemSubVolumeGroupDetails)
+
+	if err != nil {
+		r.Log.Error(
+			err,
+			"Failed to update CephFilesystemSubVolumeGroup.",
+			"CephFilesystemSubVolumeGroup",
+			klog.KRef(r.cephFilesystemSubVolumeGroup.Namespace,
+				r.cephFilesystemSubVolumeGroup.Name),
```

Comment:
As of this code base conventions, we do not break lines in this manner:
Please change to one of the following:
```
			klog.KRef(r.cephFilesystemSubVolumeGroup.Namespace, r.cephFilesystemSubVolumeGroup.Name),
```
or 
```
			klog.KRef(
				r.cephFilesystemSubVolumeGroup.Namespace,
				r.cephFilesystemSubVolumeGroup.Name,
			),
```

The main reason is that git's diff tracking is line-based. as such we either have the entire param on a single line of split the call to line per param